client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -1,37 +1,42 @@
 import { Button, Gridicon } from '@automattic/components';
+import classnames from 'classnames';
 import { translate, useRtl } from 'i18n-calypso';
 import Badge from 'calypso/components/badge';
 import { isTaskDisabled } from './task-helper';
 import { Task } from './types';
 
 const ChecklistItem = ( { task }: { task: Task } ) => {
 	const isRtl = useRtl();
-	const { id, isCompleted, actionUrl, title, actionDispatch } = task;
+	const { id, isCompleted, keepActive, actionUrl, title, actionDispatch } = task;
```

Comment:
We now have isCompeted, taskDisabled, and keepActive for each task. I wonder if we can simplify and still keep the range of conditional checks we need? Theoretically, it seems we really just need to know two things: if the is task complete, and if the task be enabled/active (regardless of completion status).

Reply:
@edanzer Initially, I saw thinking the same thing, but we can't disregard the completion status since it effects the styling of the task

Reply:
@edanzer - Good point. We might be able to move the dependencies logic to the task-helper file, then simply rely on `keepActive` and `isCompleted` for disabling/enabling task + the styling.

Going to add an example shortly.